packages/components/label/src/label.tsx

```diff
@@ -75,15 +88,15 @@ const Label = (props: TLabelProps) => {
   );
 
   return (
-    <label id={props.id} htmlFor={props.htmlFor}>
-      <Text.Body tone={props.tone} isBold={props.isBold}>
+    <label css={getLabelStyles()} id={props.id} htmlFor={props.htmlFor}>
```

Comment:
As @kark [suggested](https://github.com/commercetools/ui-kit/pull/2356/files#r1043192538), perhaps it would be simpler to use the css value here:
```suggestion
    <label
      css={css`
         > div {
           font-weight: ${designTokens.fontWeight500};
         }
      `}
      id={props.id}
      htmlFor={props.htmlFor}
    >
```

Reply:
Fixed in this commit - [9fab790](https://github.com/commercetools/ui-kit/pull/2356/commits/9fab79008b6f56b6e1f368aa32ab473ab39f55f0)